batch.py

```diff
@@ -734,7 +744,17 @@ def manage_concurrency_for_job(job: hb.job.Job):
     dependencies_dict: Dict[str, hb.job.Job] = {}
     plink_root = output_path('plink_files')
     logging.info('before glob (bim files)')
-    bim_files = list(to_path(plink_root).glob('*.bim'))
+    storage_client = storage.Client()
+    bucket = get_config()['storage']['default']['default'].replace('gs://', '')
+    prefix = f"{get_config()['workflow']['output_prefix']}/plink_files"
```

Comment:
os.path.join would be safer here if output_prefix contains a trailing slash:

```suggestion
    prefix = os.path.join(get_config()['workflow']['output_prefix'], 'plink_files')
```